app/services/api/latest_additions.rb

```diff
@@ -0,0 +1,40 @@
+module Api
+  class LatestAdditions
+    def initialize(count)
+      @count = count
+    end
+
+    def call
+      (serialize_litigations + serialize_legislations).sort_by { |item| - item[:created_at].to_time.to_i }
+    end
+
+    private
+
+    def serialize_litigations
+      Litigation.last(@count).map do |item|
```

Comment:
I think for now this logic works. we need to see once we have proper data, because we might want to filter on specific types of events instead of getting just the latest laws and litigations. but for now should be fine!

Reply:
at least those should be published ones. `Litigation.published` but that's for everything we need on frontend.